src/installer/tests/Microsoft.NET.HostModel.Tests/Microsoft.NET.HostModel.Tests.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Microsoft.NET.HostModel.Tests</Description>
+    <TargetFramework>$(TestInfraTargetFramework)</TargetFramework>
+    <AssemblyName>Microsoft.NET.HostModel.Tests</AssemblyName>
+    <PackageId>Microsoft.NET.HostModel.Tests</PackageId>
```

Comment:
Why does it need "PackageId" ?

Reply:
It shouldn't - left over from the other test project that I just renamed/edited for this.

```suggestion
```